environments/linux/docker/compose.yml

```diff
@@ -53,8 +55,9 @@ services:
       target: devenv
       args: &ubuntu18-args
         BUILDKIT_INLINE_CACHE: "1"
+        USE_LLVM_INSTALLER: 0
         ROOT_IMAGE: "nvcr.io/nvidia/cuda:11.8.0-devel-ubuntu18.04"
-        COMPILERS: "g++-5 g++-6 g++-7 clang-7"
+        COMPILERS: "g++-6 g++-7 clang-7"
```

Comment:
AFAIK we settled on gcc-7 as the minimal supported version.  

Do we want to drop gcc-6 while we are at it?

Reply:
Will consider it dropping later in another patch. In tandem with #341 would be good timing.